src/gluonnlp/data/word_embedding_evaluation.py

```diff
@@ -709,6 +758,37 @@ class GoogleAnalogyTestSet(WordAnalogyEvaluationDataset):
 
     License: Unspecified
 
+    Each sample consists of two analogical pairs of words.
+
+    Parameters
```

Comment:
@leezu some parameters are missed from the analogy classes. was it intentional? I added it for only GoogleAnalogy but not yet for BiggerAnalogy.

Reply:
The missing documentation was not intentional. Thanks for fixing!